Core/views.py

```diff
@@ -116,8 +119,44 @@ class RegistrationView(FormView):
     def post(self, request, *args, **kwargs):
         alert = ''
         form = RegistrationForm(request.POST)
+
         if form.is_valid():
+            event_receiver_email = form.cleaned_data['email']
             form.save()
+
+            allowed = False
+
+            try:
+                allowed = (Email_Content.objects.get(event=self.event)).mail_allowed
+            except:
+                pass
+
+            if allowed:
+                content = Email_Content.objects.get(event=self.event)
+                all_files = Email_Attachment.objects.filter(event=self.event)
+
+                subject = content.subject
+                message = render_to_string('event_email_template.html', {
+                    'content': content,
+                    'event': self.event,
+                })
+
+                from_mail = EMAIL_HOST_USER
+                to_mail = [event_receiver_email]
+
+                mail = EmailMessage(subject, message, from_mail, to_mail)
+                mail.content_subtype = "html"
+                mail.mixed_subtype = 'related'
+
+                image_sub_type = ((str(self.event.pic_path)).split('.'))[-1]
```

Comment:
remove redundant parenthesis
```suggestion
                image_sub_type = str(self.event.pic_path).split('.')[-1]
```